httpx/_multipart.py

```diff
@@ -158,9 +157,8 @@ def render_data(self) -> typing.Iterator[bytes]:
             yield self._data
             return
 
-        if self._consumed:  # pragma: nocover
+        if hasattr(self.file, "seek"):
```

Comment:
This is more of a question, so we do `seek` for the first file upload too, right?

Reply:
Yup